custom-metrics-stackdriver-adapter/Dockerfile

```diff
@@ -12,10 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM golang:1.13-alpine as builder
+FROM golang:1.16-alpine as builder
+ENV GO111MODULE=on
 WORKDIR ${GOPATH}/src/github.com/GoogleCloudPlatform/k8s-stackdriver/custom-metrics-stackdriver-adapter
 COPY . ./
-RUN CGO_ENABLED=0 GOOS=linux go build -mod=vendor -o /adapter
+RUN CGO_ENABLED=0 GOOS=linux go build -mod=mod -o /adapter
```

Comment:
-mod=mod means go will automatically update go.mod, I don't that's what we want. -mod=readonly seems like a better choice.

Reply:
```
In Go 1.15 and lower, the -mod=mod flag was enabled by default, so updates were performed automatically. Since Go 1.16, the go command acts as if -mod=readonly were set instead: if any changes to go.mod are needed, the go command reports an error and suggests a fix.
```
Good point. Will use **readonly**